com.openmusicplayer.OMP.json

```diff
@@ -0,0 +1,257 @@
+{
```

Comment:
Please reformat your manifest with `jq` or anything else that will produce more formatted file.

Reply:
You also need either per module or global cleanup section to delete leftovers like development headers from the final app.

Reply:
Thanks for the help! What folders exactly should I cleanup?

Reply:
> What folders exactly should I cleanup?

Usually `/app/include`, `/app/lib/pkgconfig`, `/app/share/cmake`, `/app/share/man`, also `*.a` files, sometimes `*.la` files, but be careful with the last one!
Just remember to omit the `/app` prefix in the cleanup section (so it should be `/include`, `/lib/pkgconfig`, `/share/cmake`, `/share/man`, etc.)

Example:
https://github.com/scx/flathub/blob/729d60e79597ec6f10e615ce266d1a7d0fe7cb5c/me.mitya57.ReText.yaml

Reply:
Thank you, scx. Your help is very appreciated.